.github/ISSUE_TEMPLATE/release.yaml

```diff
@@ -0,0 +1,96 @@
+name:  Release
+description: Release/Pre-release of a version.
+labels: [release]
+body:
+- type: dropdown
+  id: release_type
+  label: Release Type
+  description: which type of release is this release?
+  options:
+  - major release
+  - minor release
+  - patch release
+- type: textarea
+  id: release_version
+  attributes:
+    label: Release Version
+    description: Version of this release (e.g `2.4.0`)
+    placeholder:
+- type: textarea
```

Comment:
I used `textarea` for all parts from `RELEASE.md`, should I change some of them to other types, such as `checkbox`?

Reply:
Almost all of it, yes. Everything in the current doc is either a checkbox or a header/description for a group of them.

Reviewing one of the other forms, it looks like we'd want something like this:

```
- type: checkboxes
  attributes:
    label: Release Branch
    description: **For all releases** For this step we're going to start with the `main` branch to create our release branch (e.g. release/X.Y.Z) which will later be submitted as a pull request back to `main`.
    options:
    - label: ensure that you have up to date copy of `main`: `git fetch --all`
    - label:  create the release branch for the version (e.g. `release/1.3.1`): `git branch -m release/x.y.z`
```

Unrelated to that, could you enable issues on your fork? It'd let us test the form as we'd actually see it.

Reply:
https://github.com/randmonkey/kubernetes-ingress-controller/issues/new?assignees=&labels=release&template=release.yaml